ironfish/src/primitives/note.test.ts

```diff
@@ -0,0 +1,48 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+import { createNodeTest, useAccountFixture, useMinerBlockFixture } from '../testUtilities'
+
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+describe('Note', () => {
+  const nodeTest = createNodeTest()
+
+  it('check if a transaction is not a miners fee', async () => {
+    const nodeA = nodeTest.node
```

Comment:
The primitives tests should probably not involve an entire node and chain. I would expect them to test only the constructor and it's own data/functions. The test name seems counterintuitive to what the test is actually testing (that the miner's fee transaction is a specific sender key).  Tests related to the miner's fee would probably make more sense in the `strategy` or `createMinersFee` test files